core/arch/opnd.h

```diff
@@ -2418,13 +2418,19 @@ reg_32_to_opsz(reg_id_t reg, opnd_size_t sz);
 
 DR_API
 /**
- * Given a general-purpose register of any size, returns a register in the same
- * class of the given size.  For example, given \p DR_REG_AX or \p DR_REG_RAX
- * and \p OPSZ_1, this routine will return \p DR_REG_AL.
+ * Given a general-purpose or SIMD register of any size, returns a register in the same
```

Comment:
Not true for ARM. Please add a note that ARM is not yet supported.

Reply:
Done. We should raise another issue for ARM, or avoid closing the current one.

Reply:
Please leave the current issue open and change "Fixes" to "Issues:" in the commit message.

Reply:
Done